test/e2e/logging_log_delivery_test.go

```diff
@@ -78,11 +82,12 @@ var _ = Describe("Logging", Label("logging"), func() {
 		})
 
 		It("Should verify end-to-end log delivery with custom output", func() {
-			logsShouldBeDelivered(logProducerPodName, urls)
+			logsShouldBeDelivered(logProducerName, urls.MockBackendExport(mockDeploymentName))
 		})
 	})
 })
 
+// TODO this function is the same as deploymentShouldBeRunning except that the timeout is doubled
```

Comment:
The `logBackendShouldBeRunning` function is basically the same deploymentShouldBeRunning except that the timeout is higher. It would be nice to combine them in some way, I am still looking into this.